mocks/service_mock.go

```diff
@@ -38,10 +38,16 @@ func (m *ServiceMock) StartServer() error {
 	return nil
 }
 
+// Not used: backward compatibility with public interface
 func (m *ServiceMock) ShutdownServer() {
```

Comment:
This is not a library-wide public method, so feel free to change its signature. Simply expand it with Context argument.